### PR TITLE
Fix #313 Documenting release process

### LINKED ISSUE #313: Plugin failed to load properly on Fedora
I installed vsm plugin on fedora 23 and it throws an error 

```
naina@dhcp35-1 ~]$ vagrant status 
Vagrant failed to initialize at a very early stage:
The plugins failed to load properly. The error message given is shown below.
cannot load such file -- zip
```

The same plugin version is working fine on windows , but for fedora it is throwing error while executing commands:
vagrant version,status,global-status,up etc.This issue is blocker for fedora
vsm version  = vagrant-service-manager (1.1.0.1)

### PR DESCRIPTION
**Pull request by hferentschik**

### CODE REVIEW

**Comment by hferentschik** [comment]
@budhrg @LalatenduMohanty @navidshaikh

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -102,3 +102,33 @@ $ git merge fix-123
...
+** Move unresolved issues into an upcoming release
+** Close milestone
+* Update the link:CHANGELOG.adoc[CHANGELOG]
+* Update link:lib/vagrant-service-manager/version.rb[version.rb] with the correct
+  release version. Consider http://semver.org/:[Semantic Versioning] guidelines
+  when deciding on the release version
+* Commit _CHANGELOG_ and _version.rb_ using commit message of the form:
+  `vagrant-service-manager version=<version>`
+* Push commit to upstream
+* Release the gem: `bundle exec rake release`
```
Comment (LalatenduMohanty) [inline CONTRIBUTING.adoc:122]: We should add step to build the gem.

Comment (hferentschik) [inline CONTRIBUTING.adoc:122]: the release task takes care of this

Comment (LalatenduMohanty) [inline CONTRIBUTING.adoc:122]: Cool, I did not know that `bundle exec rake release` also builds the gem.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -102,3 +102,33 @@ $ git merge fix-123
...
 $ git push origin master
 -------------------------------------------------------------------------
+
+== Releasing
+
+* Review https://github.com/projectatomic/vagrant-service-manager/milestones[milestone] for the current release
+** Make sure all issues are resolved
+** Move unresolved issues into an upcoming release
+** Close milestone
+* Update the link:CHANGELOG.adoc[CHANGELOG]
```
Comment (brgnepal) [inline CONTRIBUTING.adoc:115]: Adding a format here for CHANGELOG update would make consistent updates.

Comment (hferentschik) [inline CONTRIBUTING.adoc:115]: Let's maybe discuss this a bit later. Seems we have not reached a consensus here.

Comment (hferentschik) [inline CONTRIBUTING.adoc:115]: For now we stick to the current format.

**Comment by LalatenduMohanty** [comment]
LGTM

**Comment by LalatenduMohanty** [comment]
cc @bexelbie

**Pull request merged**